content/en/docs/started/installing-kubeflow.md

```diff
@@ -200,12 +334,16 @@ The following table lists distributions which are <em>maintained</em> by their r
   </table>
 </div>
 
-## Raw Kubeflow Manifests
+### Raw Manifests
 
-The raw Kubeflow Manifests are aggregated by the [Manifests Working Group](https://github.com/kubeflow/community/tree/master/wg-manifests)
-and are intended to be used as the **base of packaged distributions**.
+The raw Kubeflow Manifests are aggregated by the Manifests Working Group and are intended to be
```

Comment:
```suggestion
The Kustomize Kubeflow Manifests are aggregated by the Manifests Working Group and are intended to be
```

Reply:
I am concerned about using the Raw Manifest term.   I dont believe that we use it in the Manifest docs.   Maybe just Manifests ?

Reply:
The reason we chose "raw manifests" historically is because it highlights that they are not ready for production out of the box, and are not the "official" way to deploy Kubeflow (as required by the [Manifests WG charter](https://github.com/kubeflow/community/blob/master/wg-manifests/charter.md)).

The core purpose of the manifests has always been to enable the creation of distributions that are more targeted to specific environments/users.

Obviously, some advanced users have chosen to effectively make "bespoke" distributions for their company, and that's fine but it's important for users to not think that the manifest are an "out of the box" experience.

Reply:
I do not see how the name "Kustomize manifests" (since they are actually kustomize manifests) changes that. I did not add "official way to install" or something similar.

Reply:
@juliusvonkohout I really think we should discuss this in a separate PR, where we can make a decision on that branding specifically.

The website currently calls them "raw manifests" and adding more changes to this PR will only make it take longer to merge.

Reply:
I don't think we should call it "kustomize manifests" since that's implementation detail. ICYMI, kustomize now ships with kubectl as a sub-command. 

I don't think this is blocking the PR. Maybe a separate discussion.

Reply:
Can we just call them Kubeflow Manifests @juliusvonkohout @terrytangyuan @jbottum @thesuperzapper ?
E.g.
```
The Kubeflow Manifests are aggregated by the Manifests Working Group and are intended to be used as the
base of packaged distributions.
```

Reply:
SGTM

Reply:
I am more ok with using `Kubeflow Manifests`, but we should do that change in a separate PR, to allow others to discuss.

This PR is already doing a lot, and is very hard to review for stakeholders.

Reply:
@andreyvelich @terrytangyuan I am more ok with using `Kubeflow Manifests`, but we should do that change in a separate PR, to allow others to discuss.

This PR is already doing a lot, and is very hard to review for stakeholders.

Reply:
From my point of view, the final decision will be up to the Manifest WG since they maintain this section.
I don't think renaming Raw Manifests to the Kubeflow Manifests is a big deal, tbh, but I am happy to hear other community members objections.